packages/snaps-simulation/src/options.ts

```diff
@@ -21,6 +22,7 @@ const SimulationOptionsStruct = object({
     optional(nullable(record(string(), JsonStruct))),
     null,
   ),
+  hideBalances: defaulted(optional(boolean()), false),
```

Comment:
@Mrtenz Are you fine with having this as an option in `snaps-jest`?

Reply:
How are you supposed to test an interface behavior relying on this otherwise ?

Reply:
I think we should make it configurable, just asking if we should abstract the configuration away or not have it next to more "generic" configuration options

Reply:
Judging by this, we don't actually use the option? May be an oversight when we added this hook 😅

https://github.com/MetaMask/snaps/blob/6ec1d6697ac76239504ab09bc4b97c425ab12edc/packages/snaps-simulator/src/features/simulation/sagas.ts#L132-L137

Reply:
We do allow changing it in snaps-jest, but not in the simulator 😄

Reply:
Ah, I didn't look at the file name.

We could move it to an object like `preferences` within options, but since we already support locale and currency within the options object I guess it makes sense to add it here.